chia/timelord/timelord.py

```diff
@@ -183,25 +183,22 @@ async def _handle_client(self, reader: asyncio.StreamReader, writer: asyncio.Str
 
     async def _stop_chain(self, chain: Chain):
         try:
-            while chain not in self.allows_iters:
```

Comment:
Why are we removing the while loop?

Reply:
to avoid the infinite loop that happens in rare cases (written in the description).